packages/mdc-button/_mixins.scss

```diff
@@ -66,6 +67,24 @@
   border-radius: $corner-radius;
 }
 
+@mixin mdc-button-angled-corner($background-color, $top-left-size, $top-right-size: $top-left-size, $bottom-right-size: $top-left-size, $bottom-left-size: $top-right-size, $stroke-width: 0) {
+  $threshold: $mdc-button-contained-horizontal-padding * 3 / 4;
```

Comment:
How did you arrive at 3/4? It looks better when you have that there. What that more of a guess/check?

Reply:
It was a bit of a guess / check. I started with `/ 2`, but that made the buttons look too wide. I wanted the value to be relative to the overall default padding, though.

...although this makes me realize that this should probably also depend on whether the horizontal-padding mixin is being used, which means yet another parameter to add. Not sure how I feel about this...

Reply:
Gotcha...I tried a few different values for `$mdc-button-contained-horizontal-padding` and overall looks good. 

RE: more arguments: I know with JS we can pass an object. Can we do the same in Sass?

Reply:
We can, and OTOH Sass also supports named parameters. My concern was just parameter overload either way.